tests/DCPS/Compiler/anonymous_types/test.idl

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Add the line `#include <tao/LongSeq.pidl>` so the generated code can compile on Visual C++.